charts/jobs/templates/cronjob.yaml

```diff
@@ -1,15 +1,17 @@
 {{- $v := .Values }}
 {{- $teamSecrets := (include "itemsByName" ($v.teamSecrets | default list) | fromYaml) }}
+{{- $podSecurityContext := get $v "podSecurityContext" | default (dict "runAsNonRoot" true "runAsUser" 1001 "runAsGroup" 1001) }}
```

Comment:
The only thing I would like to add is what @Morriz and I discussed: to merge job.yaml and cronjob.yaml to avoid this kind of duplication... (same line in `job.yaml`). But I guess it's out of scope, unless we can define a helper function?

Reply:
It could be done to make it more DRY but it will lead to a lot of if then elses. Lets skip that for now